setup.sh

```diff
@@ -53,4 +53,6 @@ echo "Here is a suggestion: in order to execute \"topoana.exe\" without any path
 echo "1) In case of bash, please copy the following statement and paste it into ~/.bash_profile,"
 echo "   export PATH=${TopoAna}/bin:\$PATH"
 echo "2) In case of (t)csh, please copy the following statement and paste it into ~/.(t)cshrc,"
-echo "   setenv PATH ${TopoAna}/bin:\$PATH"
\ No newline at end of file
+echo "   setenv PATH ${TopoAna}/bin:\$PATH"
+
+source "utilities/Functions.sh"
```

Comment:
In my opinion, it would be better to remove the last statement, and change the export and setenv as follows:
`echo "   export PATH=${TopoAna}/utilities:${TopoAna}/bin:\$PATH"`
`echo "   setenv PATH ${TopoAna}/utilities:${TopoAna}/bin:\$PATH"`
In such a way, users do need to `source utilities/Fuctions.sh` when they start a new session.

Reply:
Maybe better:

1. Change `setup.sh` into `install.sh` that is only responsible for the compiling (whether through `make` or `g++`).

2. Make a new setup `setup.sh` file that sets all parameters in `bash`, containing:
```
export TopoAna="$(cd "$(dirname "${BASH_SOURCE[0]}" )" && pwd)"
export PATH=${TopoAna}/bin:\$PATH
source ${TopoAna}/utilities/Functions.sh
```

3. Have the user add the line
```
source /<path to topoana>/setup.sh
```
to `bashrc` or `bash_profile`.

Reply:
Yes, it's a nice suggestion. I will try to do that.